doc/tor.md

```diff
@@ -88,8 +89,8 @@ for normal IPv4/IPv6 communication, use:
 
 	./bitcoin -onion=127.0.0.1:9050 -externalip=57qr3yd1nyntf5k.onion -discover
 
-3. Automatically listen on Tor
---------------------------------
+<a name="3-automatically-listen-on-tor"></a>
```

Comment:
Unnecessary, remove?

Reply:
I believe that the section titles are relevant. They have been converted from the alternate style headings to the standard style headings since the alternate style doesn't work with numbered heading items. Not all MD rendering versions creates named anchors for titles so, the named anchor is included additionally, not instead.

Reply:
> Not all MD rendering versions creates named anchors for titles

I don't agree in adding redundant anchors because other MD renderers don't support it. I think we review MD based on what GH supports.

Reply:
Last time I checked GH didn't but, now [certainly does](view-source:https://github.com/Willtech/bitcoin/blob/8dcd724f4d8719a645606932e2a9289b521989a3/doc/tor.md). I will remove the anchor and recommit.